test.js

```diff
@@ -862,55 +862,55 @@ test('emit error on EAGAIN (sync)', (t) => {
   stream.on('finish', () => {
     fs.readFile(dest, 'utf8', (err, data) => {
       t.error(err)
-      t.equal(data, 'hello world\nsomething else\n')
+      t.equal(data, 'something else\n')
```

Comment:
@mcollina this is suspicious. I'm unsure what is correct here. If the operation failed... then I don't think it should have the data?

Reply:
It should, that's the point of the test